front/tests/010-op-route-tab.spec.ts

```diff
@@ -0,0 +1,240 @@
+    await operationalStudiesPage.verifyNoSelectedRoute(selectedLanguage);
+    await operationalStudiesPage.performPathfindingByTrigram('WS', 'NES', 'MES');
+
+    // Define expected map markers values
+    const expectedMapMarkersValues = ['West_station', 'North_East_station', 'Mid_East_station'];
+
+    // Verify map markers in Chromium browser
+    if (browserName === 'chromium') {
+      await operationalStudiesPage.verifyMapMarkers(...expectedMapMarkersValues);
+    }
+
+    // Verify absence of tab warning
+    await operationalStudiesPage.verifyTabWarningAbsence();
+  });
+
+  test('should correctly add waypoints in a route for operational study', async ({
+    page,
+    browserName,
+  }) => {
+    const operationalStudiesPage = new OperationalStudiesPage(page);
+    const scenarioPage = new ScenarioPage(page);
+    const rollingStockSelector = new RollingStockSelectorPage(page);
+
+    // TODO: DROP TSV1: remove this part
+    const homePage = new HomePage(page);
+    await homePage.goToHomePage();
+    await homePage.toggleTSV2();
+
+    // Navigate to the created scenario page
+    await page.goto(
+      `/operational-studies/projects/${project.id}/studies/${study.id}/scenarios/${scenario.id}`
+    );
+
+    // Click on add train button
+    await operationalStudiesPage.clickOnAddScenarioTrainBtn();
+
+    // Open Rolling Stock Selector, search for the added train, and select it
+    await operationalStudiesPage.openEmptyRollingStockSelector();
+    await rollingStockSelector.searchRollingstock(electricRollingStockName);
+
+    const electricRollingstockCard = rollingStockSelector.getRollingstockCardByTestID(
+      `rollingstock-${electricRollingStockName}`
+    );
+    await electricRollingstockCard.click();
+    await electricRollingstockCard.locator('button').click();
+
+    // Verify selected rolling stock name
+    expect(await rollingStockSelector.getSelectedRollingStockName.innerText()).toEqual(
+      electricRollingStockName
+    );
+
+    // Perform pathfinding
+    await scenarioPage.openTabByDataId('tab-pathfinding');
+    await operationalStudiesPage.performPathfindingByTrigram('WS', 'NES');
+
+    // Define expected waypoints and their values
+    const expectedViaValues = [
+      { name: 'Mid_West_station', ch: 'BV', uic: '3', km: 'KM 11.850' },
+      { name: 'Mid_East_station', ch: 'BV', uic: '4', km: 'KM 26.300' },
+    ];
+
+    // Add new waypoints
+    await operationalStudiesPage.addNewWaypoints(
+      2,
+      ['Mid_West_station', 'Mid_East_station'],
+      expectedViaValues
+    );
+
+    // Define expected map markers values
+    const expectedMapMarkersValues = [
+      'West_station',
+      'Mid_West_station',
+      'Mid_East_station',
+      'North_East_station',
+    ];
+
+    // Verify map markers in Chromium browser
+    if (browserName === 'chromium') {
+      await operationalStudiesPage.verifyMapMarkers(...expectedMapMarkersValues);
+    }
```

Comment:
Like previous comment
```suggestion
     // Verify map markers in Chromium browser
    if (browserName === 'chromium') {
      const expectedMapMarkersValues = [
        'West_station',
        'Mid_West_station',
        'Mid_East_station',
        'North_East_station',
      ];
      await operationalStudiesPage.verifyMapMarkers(...expectedMapMarkersValues);
    }
```

Reply:
The `const` this time is used in the `reverse() `function in the next if so it should stay like that

Reply:
You're right :+1: